.gitignore

```diff
@@ -2,3 +2,4 @@ target
 project/target
 .idea
 .idea_modules
+.bsp
```

Comment:
Generally speaking we are fine with merging `.gitignore` changes, but regardless, I'll say: you'll probably want to add `.bsp` to your global gitignore, so then you won't need to submit this change to every repository that you contribute to.